README.md

```diff
@@ -301,6 +302,31 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+#### grcov with Gitlab
+
+Here is an example `.gitlab-ci.yml` which will build and collect coverage data in Gitlab:
+
+```yaml
+build:
+  script:
+    - cargo test --workspace
+    # Optionally, run some other command that exercises your code:
+    - LLVM_PROFILE_FILE="target/coverage/%p-%m.profraw" ./bin/integration-tests --foo bar
+    # Once a release with https://github.com/mozilla/grcov/pull/893 is available, modify this to use the multiple-outputs mechanism :)
+    - grcov target/coverage --binary-path target/debug -s . -o target/coverage.xml -t cobertura
+    - grcov target/coverage --binary-path target/debug -s . -o target/coverage -t html
+    # This is an awful hack whose goal is to print the coverage number by extracting it from the HTML report.
+    - awk 'c&&--c { gsub(" *<abbr[^>]+>", "Coverage: ", $0); print; } /<p class="heading">Lines</p>/{c=2}' target/coverage/index.html
```

Comment:
Might be easier to extract from the markdown output?

Reply:
Yes, it might be. `grcov` takes about 45 seconds per run on my project, though, so it's a noticeable slowdown to have to do even 2 runs of it. Thus, I used the HTML report to get the coverage rather than running another time. Eagerly awaiting a release with multi-output mode :)

The xml output is also probably easier to get the overall coverage from, but in my testing it looks like you'd want `--keep-only 'src/*'` or similar on that invocation to avoid getting a bunch of un-covered external libraries included in your coverage report. I can switch to that if you like, this MR was just assembled from what I'm actually doing, not necessarily thinking about the best thing to do.

I'll read over the changes with "what's the best way to do this" in mind.